### PR TITLE
To Do List

### PR DESCRIPTION
**Pull request by Fayob**
## TO DO LIST TASK

- setup linters
- set up webpack
- ignore file in gitignore
- add mark up file
- add styles file
- add javascript file

### CODE REVIEW

File: src/modules/edit_task.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  const element = document.querySelectorAll('.text');
+  element.forEach((ele) => {
+    ele.addEventListener('click', () => {
+      ele.contentEditable = true;
+      //   ele.style.backgroundColor = 'lightyellow';
+      ele.parentElement.parentElement.lastElementChild.previousElementSibling.classList.add('hidden');
+      ele.parentElement.parentElement.lastElementChild.classList.remove('hidden');
+    });
+  });
+};
```
Comment (m05e5) [inline src/modules/edit_task.js]: - [ ] Well done with the edit. As mentioned in the requirements,  all changes in your todos should be saved in local storage. kindly save the todo in local storage after exit

File: src/modules/add_task.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      if (!input.value) {
+        alert('Please enter a task');
+        return;
+      }
+      const eachTask = {
+        description: input.value,
+        completed: false,
+        index: tasks.length + 1,
+      };
+
```
Comment (m05e5) [inline src/modules/add_task.js:17]: - [ ] Actually when we add some todos, delete one and add another one as you can see some todos comes out with the same index. This creates issues when we now delete one of the todos with same index. please kindly consider and fix it
image

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Fayob  , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please see comments bellow

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Fayob, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**